daemons/controld/controld_schedulerd.c

```diff
@@ -478,10 +478,9 @@ do_pe_invoke_callback(xmlNode * msg, int call_id, int rc, xmlNode * output, void
     } else if (num_cib_op_callbacks() > 1) {
         crm_debug("Re-asking for the CIB: %d other peer updates still pending",
                   (num_cib_op_callbacks() - 1));
-        sleep(1);
-        timer = mainloop_timer_add("timer", 1000, TRUE, sleep_timer, NULL);
+        timer = mainloop_timer_add("timer", 1000, FALSE, sleep_timer, NULL);
         mainloop_timer_start(sleep_timer);
-        return;
+        return G_SOURCE_REMOVE;
     }
 
     CRM_LOG_ASSERT(output != NULL);
```

Comment:
Please squash this commit together with the first one (see https://git-scm.com/docs/git-rebase#_interactive_mode), give it a good commit message that references the task you're fixing, and make sure that this PR also has that title.  Thanks!